documentation/modules/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -7,7 +7,31 @@
 
```

Comment:
Suggestion for title: _Kafka cluster resources created by the Cluster Operator_

Reply:
Chapter with similar title (_ List of resources created as part of Kafka cluster_) is in all chapters (Connect, Mirror Maker etc.). I do not think we should make just this single one different.